epilepsy12/views/case_views.py

```diff
@@ -641,13 +661,15 @@ def update_case(request, organisation_id, case_id):
     """
     Django function based view. Receives POST request to update view or delete
     """
-    case = get_object_or_404(Case, pk=case_id)
-    form = CaseForm(instance=case)
-
     organisation = Organisation.objects.filter(pk=organisation_id).get()
+    case = get_object_or_404(Case, pk=case_id)
+    form = CaseForm(instance=case, organisation_id=organisation_id)
 
     # set select boxes for situations when postcode unknown
-    country_choice = ("ZZ993CZ", "Address unspecified - England")
+    country_choice = (
+        "ZZ993CZ",
+        "Address unspecified - England",
+    )  # TODO we are using the same value for England and Jersey. is that correct?
```

Comment:
I guess not but I'm not sure it's worth the conditional logic?

@nikyraja @SAbdin93 Should we add a new ZZ code for unspecified addresses in Jersey?

Reply:
The ZZ codes are [NHS standards](https://digital.nhs.uk/data-and-information/data-collections-and-data-sets/data-sets/mental-health-services-data-set/submission-guidance/make-a-basic-submission#:~:text=If%20the%20postcode%20of%20the,Postcode%20is%20unknown%20%2D%20ZZ99%203WZ).  Jersey are not part of the NHS so I guess will not have any codes. NHS Wales have their own versions of the same but different codes. Maybe there is a code for people who live on yachts?